lms/templates/courseware/course_about.html

```diff
@@ -294,6 +294,7 @@ <h1>
             <li class="important-dates-item"><span class="icon fa fa-book" aria-hidden="true"></span><p class="important-dates-item-title">${_("Requirements")}</p><span class="important-dates-item-text prerequisites">${get_course_about_section(request, course, "prerequisites")}</span></li>
           % endif
         </ol>
+        <%static:optional_include_mako file="course_about_sidebar_extra.html" />
```

Comment:
this is actually a different feature, and has already been separately merged upstream.